operator/cmd/mesh/operator-remove.go

```diff
@@ -101,32 +100,7 @@ func operatorRemove(args *rootArgs, orArgs *operatorRemoveArgs, l clog.Logger, d
 	l.LogAndPrint("\n*** Success. ***\n")
 }
 
-func deleteManifest(manifestStr, componentName string, opts *kubectlcmd.Options, l clog.Logger) bool {
-	l.LogAndPrintf("Deleting manifest for component %s...", componentName)
-	objs, err := object.ParseK8sObjectsFromYAMLManifest(manifestStr)
-	if err != nil {
-		l.LogAndPrint("Parse error: ", err, "\n")
-		return false
-	}
-	stdout, stderr, err := kubectlcmd.New().Delete(manifestStr, opts)
-
-	success := true
-	if err != nil {
-		cs := fmt.Sprintf("Component %s delete returned the following errors:", componentName)
-		l.LogAndPrintf("\n%s\n%s", cs, strings.Repeat("=", len(cs)))
-		l.LogAndPrint("Error: ", err, "\n")
-		success = false
-	} else {
-		l.LogAndPrintf("Component %s deleted successfully.", componentName)
-		if opts.Verbose {
-			l.LogAndPrintf("The following parseObjectSetFromManifest were deleted:\n%s", k8sObjectsString(objs))
-		}
-	}
-
-	if !ignoreError(stderr) {
-		l.LogAndPrint("Error detail:\n", stderr, "\n")
-		l.LogAndPrint(stdout, "\n")
-		success = false
-	}
-	return success
+func deleteManifest(_, _ string, _ *Options, l clog.Logger) bool {
+	l.LogAndError("Deleting manifest not implemented")
```

Comment:
if its not implemented should we just delete the function?

Reply:
this is todo, i didn't want to tackle in this pr but it's needed.